go/gramework/main.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"github.com/gramework/gramework"
+	"fmt"
+)
+
+func main() {
+	app := gramework.New()
+
+	app.GET("/", func(ctx *gramework.Context) error {
```

Comment:
actually, in Gramework you can literally

```go
app.GET("/", "")
```

:)

Reply:
:heart: